pom.xml

```diff
@@ -24,6 +24,7 @@
     <changelist>999999-SNAPSHOT</changelist>
     <jenkins.version>2.289.1</jenkins.version>
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
+    <useBeta>true</useBeta>
```

Comment:
Is this really required?

Reply:
Yes. To see the reason why, try compiling with it removed.

Reply:
I nagged Daniel to remove the scary beta warning in https://github.com/jenkinsci/jenkins/pull/4501#issuecomment-1111098405.